Gem/Code/CMakeLists.txt

```diff
@@ -69,31 +69,10 @@ ly_create_alias(NAME MultiplayerSample.Servers  NAMESPACE Gem TARGETS Gem::Multi
 # Gem dependencies
 ################################################################################
 
-# The GameLauncher uses "Clients" gem variants:
-ly_enable_gems(PROJECT_NAME MultiplayerSample GEM_FILE enabled_gems.cmake
-    TARGETS MultiplayerSample.GameLauncher
-    VARIANTS Clients)
+# Maps tthe MultiplayerSample Project with the specified list of enabled gems
+ly_enable_gems(PROJECT_NAME MultiplayerSample GEM_FILE enabled_gems.cmake)
 
 # If we build a server, then apply the gems to the server
```

Comment:
Do we need to update this comment also?

Reply:
Yeah that I can up the comment around applying gems to the server